charts/kube-prometheus-stack/Chart.lock

```diff
@@ -1,12 +1,12 @@
 dependencies:
 - name: kube-state-metrics
   repository: https://prometheus-community.github.io/helm-charts
-  version: 4.15.0
+  version: 4.18.0
 - name: prometheus-node-exporter
   repository: https://prometheus-community.github.io/helm-charts
   version: 3.3.1
```

Comment:
do you want to update node exporter as well https://github.com/prometheus-community/helm-charts/commits/main/charts/prometheus-node-exporter

Reply:
done